File: Back/petPlace/src/main/java/com/minjeok4go/petplace/chat/dto/ChatMessageDTO.java
package com.minjeok4go.petplace.chat.dto;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDateTime;
import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class ChatMessageDTO {
    private Long chatId;
    private Long chatRoomId;
    private Long userId;
    private String nickname;
    private String message;
    private List<String> imageUrls; // 여러 이미지 URL
    private LocalDateTime createdAt;
}

File: Front/PetPlace/app/src/main/java/com/example/petplace/data/model/chat/ChatMessageDTO.kt
package com.example.petplace.data.model.chat

import com.google.gson.annotations.SerializedName

data class ChatMessageDTO(
    @SerializedName("chatRoomId")
    val chatRoomId: Long,

    @SerializedName("userId")
    val userId: Long,

    @SerializedName("message")
    val message: String,

    @SerializedName("imageUrls")
    val imageUrls: List<String> = emptyList(),

    @SerializedName("createdAt")
    val createdAt: String? = null,

    @SerializedName("chatId")
    val chatId: Long? = null
)